YMChat/src/main/assets/index.html

```diff
@@ -65,8 +65,7 @@
         window.ymConfig = {
             bot: botId || '',
             pwa: true,
-            sdk: true, // Currently only used for analytics
-            enableHistory: (enableHistory === 'true') ,
+            sdk: true,
```

Comment:
Along with this you can remove enableHistory variable on line no 46

Reply:
Done